UI/window-basic-main.cpp

```diff
@@ -4683,15 +4659,22 @@ void OBSBasic::on_actionShowMissingFiles_triggered()
 	obs_enum_all_sources(cb_transitions, files);
 
 	if (obs_missing_files_count(files) > 0) {
-		missDialog = new OBSMissingFiles(files, this);
-		missDialog->setAttribute(Qt::WA_DeleteOnClose, true);
-		missDialog->show();
-		missDialog->raise();
+		/* the window hasn't fully initialized by this point on macOS,
+		* so put this at the end of the current task queue. Fixes a
```

Comment:
This is only true on OBS' first load and not when the dialog is manually called, I feel like the comment should reflect that